.github/workflows/preview-release.yml

```
@@ -33,8 +33,14 @@ env:
 
 jobs:
   preview:
-    name: Publish preview release
+    if: ${{ github.repository_owner == 'withastro' && github.event.issue.pull_request && (contains(github.event.comment.body, '!preview')) }}
```

```suggestion
    if: ${{ github.repository_owner == 'withastro' && github.event.issue.pull_request && (startsWith(github.event.comment.body, '!preview')) }}
```

should we only allow commands with start with? which is the same as how `!bench` works